src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -0,0 +1,59 @@
+import _ from 'lodash/fp';
+
+import * as currentOrPastDate from '../../common/schemaform/definitions/currentOrPastDate';
+import * as fullName from '../../common/schemaform/definitions/fullName';
+import * as ssn from '../../common/schemaform/definitions/ssn';
+
+import { relationshipLabels, genderLabels } from '../utils/helpers';
+
+const defaultProps = [
+  'relativeFullName',
+  'relativeSocialSecurityNumber',
+  'relativeDateOfBirth',
+  'gender',
+  'relationship'
+];
+
+export default function applicantInformation(
+  schema,
+  fields = defaultProps,
+  required = ['relativeFullName']) {
+  const possibleProperties = {
+    relativeFullName: fullName.schema,
+    relativeSocialSecurityNumber: ssn.schema,
+    relativeDateOfBirth: currentOrPastDate.schema,
```

Comment:
Based on what we talked about before, I think we want to move towards getting rid of the schema being included in our common definition files, if it's not necessary (which it isn't for those three; they just have specific uiSchema objects). So I've been pushing to stop using them in PRs, but haven't updated the files themselves yet.

That includes currentOrPastDate; all the dates have the same schema (the same one as on the backend), only the uiSchemas are different.

I'm not sure what you mean about the properties vs definitions thing. I think both schemas have a common `fullName` definition and several specific fields that reference it. I think you can just use the version from `definitions` here.